scripts/trading_strategy_helpers.js

```diff
@@ -100,6 +100,10 @@ const maxUINT = new BN(2).pow(new BN(256)).sub(new BN(1))
  */
 
 const formatAmount = function(amount, token) {
+  if (amount < 1) {
+    amount = amount * 10 ** token.decimals
+    return new BN(amount.toString())
+  }
   return new BN(10).pow(new BN(token.decimals)).muln(amount)
 }
```

Comment:
In another PR I'm writing a (hopefully) robust function to format amounts, this function could be easily replaced.

Reply:
I reused your the function now.